mimic/test/test_server_images.py

```diff
@@ -112,3 +114,34 @@ def test_get_image_list_after_list_images(self):
                              get_server_image_response_body['image']['id'])
             self.assertEqual(each_image['name'],
                              get_server_image_response_body['image']['name'])
+
+    def test_get_OnMetal_server_image(self):
+        """
+        Test to verify :func:`get_image` on ``GET /v2.0/<tenant_id>/images/<image_id>``
+        """
+        helper = APIMockHelper(self, [NovaApi(['IAD'])])
+        root = helper.root
+        uri = helper.uri
+        response, body = self.successResultOf(json_request(
+            self, root, "GET", uri + '/images/detail'))
+        self.assertEqual(200, response.code)
+        images = body['images']
+        for image in images:
+            if image['metadata']['flavor_classes'] == 'onmetal':
+                onmetal_id = image['id']
+                break
+        response, body = self.successResultOf(json_request(
+            self, root, "GET", uri + '/images/' + onmetal_id))
+        self.assertEqual(200, response.code)
+        self.assertEqual(body['image']['id'], onmetal_id)
+        self.assertEqual(body['image']['metadata']['flavor_classes'], 'onmetal')
+
+    def test_OnMetal_image_list(self):
+        helper = APIMockHelper(self, [NovaApi(['IAD'])])
```

Comment:
How about adding a negative test to verify that ORD does not have onmetal images.